languages/mondu-en_US.po

```diff
@@ -0,0 +1,345 @@
+msgid ""
```

Comment:
**question:** I did not understand why you needed to add an en_US translation file. Can you explain me?

Reply:
The translation key got too large for those textst, 

We would have to do something like this...
__("This invoice was created in accordance with the general terms and conditions of <strong>%s</strong> and <strong>Mondu GmbH</strong> for the purchase on account payment model.<br/><br/>Since you have chosen the payment method to purchase on account with payment via SEPA direct debit through Mondu, the invoice amount will be debited from your bank account on the due date.<br/><br/>Before the amount is debited from your account, you will receive notice of the direct debit. Kindly make sure you have sufficient funds in your account.")

Reply:
Okay, I got your point. We can leave it for now and maybe in the future we think about what to do.